Searches/BreadthFirstSearch.java

```diff
@@ -0,0 +1,97 @@
+package Searches;
```

Comment:
Please reuse the Node class from the DFS

Reply:
Thank you for the hint. I adjusted the importing for the Node class.